x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/utils.ts

```diff
@@ -18,3 +18,23 @@ export const parsedPidOrEntityIdParameter = (parameters: {
     entity_id: parameters?.entityId?.[0] ?? '',
   };
 };
+
+const convertToMilliseconds = (value: number, unit: 'h' | 'm' | 's'): number => {
+  const unitToMilliseconds = {
+    h: (): number => value * 60 * 60 * 1000,
+    m: (): number => value * 60 * 1000,
+    s: (): number => value * 1000,
+  };
```

Comment:
can you break this out into a `const`? no need to re-define this every time the utility is used.

Reply:
875fc9470896070caf39bdab0c6ba5c29f94d78f